### PR TITLE
[CMLG-015] Implement the basic functionality of the table pagination

### PR DESCRIPTION
**Pull request by annithinggoes**
Issue:
When user only wants to view only 10 results per page, needs a method to view the next 10 results.

Solution:
Add Pagination which allows user to get more results.
Pagination not visible if only a single page of results
(To test change the url fetch to the dev backend)

Risk/Note: CSS and caching to be implemented still.

Reviewed by Emily, Kevin, Eileen, Yujia, Dave

### CODE REVIEW

File: src/components/Pagination.js [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+                // handle: [1] [2] (3) [4] (5) ...[9]
+                case ( !hasLeftElipsis && hasRightElipsis ): {
+                    const pages = range( 1, centreNumberBlocks );
+                    displayedPages = [...pages, ELLIPSIS, totalPages];
+                    break;
+                }
+
+                // handle: [1] ... [3] [4] (5) [6] [7] ...[9]
+                case ( hasLeftElipsis && hasRightElipsis ):
```
Comment (zyan225) [inline src/components/Pagination.js:91]: do we need to have the case here since it's just default behaviour?

File: src/components/Pagination.js [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+        return [LEFT_PAGE, ...displayedPages, RIGHT_PAGE];
+    }
+
+    render() {
+        const pages = this.getPageNumbers();
+
+        return (
+            <nav aria-label="Pagination">
+                <ul className="pagination">
+                    {pages.map( ( page, index ) => {
```
Comment (zyan225) [inline src/components/Pagination.js:108]: space

File: src/components/Pagination.js [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+
+                        if ( page === LEFT_PAGE ) return (
+                            <li key={ index } className="page-item">
```
Comment (zyan225) [inline src/components/Pagination.js:111]: space around "="

**Review by zyan225 — COMMENTED** [review]
Great work!
Get space back. :)

**Review by ywu666 — APPROVED** [review]
Good

**Review by EileenSiqi — APPROVED** [review]

**Pull request merged**